README.md

```diff
@@ -1,19 +1,20 @@
-# Blaze 4D **Early Development**
+![blaze](https://user-images.githubusercontent.com/68126718/125143247-71be4580-e0f0-11eb-88bc-070eb2838435.png)
 
 ## Information 
-Blaze4D is a fabric mod that changes Minecraft's rendering engine to use the Vulkan Graphics Library.\
-Blaze4D is currently in Early Development and should **NOT** be used if you do **NOT** have somewhat experience in building Fabric mods.\
+Blaze4D is a Fabric mod that changes Minecraft's rendering engine to use the Vulkan Graphics Library, it is currently in Early Development and is **NOT** intended for use by the general public.\
 Support for Blaze4D **WILL NOT** be given as the mod is in its **Early Development** stages.\
-Blaze4D is compatible with Sodium.
```

Comment:
Don't remove this line. It's factually true, not just hydos' opinion.

Reply:
Okay, sorry, I agree with you.

Reply:
Depends on how far you want to spread out 'it works'

Reply:
I'd say just about as long as combining the two doesn't cause issues that either one by itself does, they "work" together. (You can't expect much from Blaze4D at this stage, but as long as adding Sodium doesn't break it *more*, it's good.
And hydos has done his best to ensure a minimum amount of compatibility between the two up to this point. (I.E. Making sure it doesn't crash the game.)